### ISSUE TITLE
Android build failed: Try correcting the name to the name of an existing method, or defining a method named 'shader'.

### ISSUE THREAD

**Issue by RoyalCoder88** [T99Rots/flutter_metaballs#4]
Hi I updated the flutter to master channel and now when I'm trying to build for Android I get this error:
```
Launching lib\main.dart on SM A908B in debug mode...
../../../AppData/Local/Pub/Cache/hosted/pub.dev/metaballs-1.4.2/lib/native_metaballs_renderer.dart:91:37: Error: The method 'shader' isn't defined for the class 'FragmentProgram'.
 - 'FragmentProgram' is from 'dart:ui'.
Try correcting the name to the name of an existing method, or defining a method named 'shader'.
              return snapshot.data!.shader(
                                    ^^^^^^
../../../AppData/Local/Pub/Cache/hosted/pub.dev/metaballs-1.4.2/lib/metaballs_shader_sprv.dart:11:21: Error: Member not found: 'FragmentProgram.compile'.
    FragmentProgram.compile(
                    ^^^^^^^
```
![Screenshot 2022-11-17 125725](https://user-images.githubusercontent.com/6507433/202452725-8d560815-60a3-4df3-a9fd-14c653b2187b.png)

**Comment by onstonboy** [comment]
Hi @RoyalCoder88 , did you resolve that issue?

**Comment by RoyalCoder88** [comment]
Nope, :( unfortunately not yet

**Comment by T99Rots** [comment]
Hey, i hadn't noticed this issue since it was already closed, are you still running in to this issue? If so can you provide me with your flutter version you are trying to use this package with? I'm fairly certain it might be that you are using too old of a version since shader support is something fairly recently added.

**Comment by RoyalCoder88** [comment]
Hi @T99Rots now it's fixed with the version **metaballs 1.5.0**  . Thanks!

**Comment by T99Rots** [comment]
Awesome! Happy to hear!
